services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -253,6 +271,86 @@ async def _get_default_pricing_and_hardware_info(
     )
 
 
+_RAM_SAFE_MARGIN_RATIO: Final[
+    float
+] = 0.1  # NOTE: machines always have less available RAM than advertised
```

Comment:
Is this a percent? If yes maybe reflect that in the name. If a ration, can we have a hint about the ration between what?

Reply:
it is not a percent, it is a ratio